bin/util.js

```diff
@@ -74,6 +76,30 @@ module.exports = {
             return _.split(list, ',');
         },
 
+        /**
+         * Helper to parse variadic parameters for reporters to an array.
+         *
+         * eg. item1,item2
+         *
+         * @param {String} list - The comma separated parameters.
+         * @returns {String[]} - [item1, item2] - The array representation of the passed parameters.
+         */
+        reporterParse: (list) => {
+            list.trim();
+            if (list.includes(',')) {
+                list.split(',').forEach((name) => {
+                    if (name.length) {
```

Comment:
The `cli` arg gets added twice in this case. Just a minor suggestion.

```diff
- if (name.length) {
+ if (name.length && name !== 'cli') {
```

Reply:
Handling the previous edge case, this is important. The reporter does take care of checking duplicates before running it, so this isn't an issue.